cliupdate.php

```diff
@@ -1,4 +1,5 @@
 <?php
+set_time_limit(300);
```

Comment:
Why this time limit? We are already using a greater one in the `ContentLoader`.

Reply:
fix in 54235f84fd374354beb3e80e68d3ecca0590206d